core/server/models/settings.js

```diff
@@ -92,7 +92,7 @@ Settings = ghostBookshelf.Model.extend({
 
     defaults: function defaults() {
         return {
-            type: 'core'
+            type: ''
```

Comment:
Does this default ever gets executed? If it's not used I think we could remove the `defaults` function completely?